dmapiclient/search.py

```diff
@@ -136,6 +144,8 @@ def search_services_from_url(self, search_api_url, id_only=False, page=None):
 
     search_services_from_url_iter = make_iter_method('search_services_from_url', 'services')
 
-    def aggregate_services(self, index, q=None, aggregations=[], **filters):
-        response = self._get(self.get_url(path='aggregations', index=index, q=q, aggregations=aggregations, **filters))
+    def aggregate_docs(self, index, doc_type, q=None, aggregations=[], **filters):
```

Comment:
same note here about arg order.

Reply:
This is a breaking change because of the change in method name though? And the new argument is at the end of the positional args anyway?

Reply:
I am clearly unable to read.

Reply:
(but also, "keyword" args *can* be called positionally)

Reply:
(but yes, of course, moving your arg to the end wouldn't allow it to be required)